src/command_modules/azure-cli-interactive/tests/test_telemetry.py

```diff
@@ -0,0 +1,25 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import unittest
+from azclishell.telemetry import scrub
+
+
+class TelemetryTest(unittest.TestCase):
+
+    def test_scrubbing(self):
+        """ tests the scrubbing of parameters """
+        text1 = 'this is a --secret value wut wut'
+        text2 = 'this has NO secret value'
+        text3 = 'this is -my value -is alice --bob knows'
+        scrub_val = '*****'
+        self.assertEqual(scrub(text1), 'this is a --secret {} wut wut'.format(scrub_val))
```

Comment:
Based on how the CLI works, I would expected:

`--secret value wut wut`
to become
`--secret ***** ***** *****`

When you use nargs='+', scrubbing only the first item in a list is probably... not good. :)